.gitignore

```diff
@@ -25,3 +25,32 @@ _testmain.go
 
 vendor
 .cover
+
+# MacOS
+
+*.DS_Store
+.AppleDouble
+.LSOverride
+
+# Icon must end with two \r
+Icon
+
```

Comment:
Why we need to ignore MacOS file. I think just `.DS_Store` should be included in ignore list.